### ISSUE TITLE
Kapitan

### ISSUE THREAD

**Issue by Smil3ytjuuhh** [gta-chaos-mod/ChaosModV#948]
Spawns Kosatka on top of player

Player is invincible for a second so they do not die

Spawns compagnion Pavel
Gives Wrench

**Comment by anto4440** [comment]
No... this is way too random.

**Comment by Smil3ytjuuhh** [comment]
That's.. What chaos means

**Comment by anto4440** [comment]
No, i mean its too random in any situation... It wont be a funny or interesting effect.

**Comment by Smil3ytjuuhh** [comment]
It is possible to make it into 2 effects i think, spawn compagnion pavel and spawn submarine

**Comment by anto4440** [comment]
spawn companionn pavel has already been requested...


Den tir. 6. apr. 2021 kl. 10.04 skrev Smil3ytjuuhh ***@***.***
